sdk/storage/Azure.Storage.DataMovement/src/Shared/TransferManager.cs

```diff
@@ -260,6 +261,32 @@ public virtual Task<bool> TryPauseTransferAsync(string transferId, CancellationT
             }
         }
 
+        /// <summary>
+        /// Gets the current transfers stored in the <see cref="TransferManager"/>.
+        /// </summary>
+        /// <param name="filterByStatus">
+        /// If specified, the returned list of transfers will have only have the transfers
+        /// of which match the status specified.
+        ///
+        /// If not specified or specified to <see cref="StorageTransferStatus.None"/>,
+        /// all transfers will be returned regardless of status.
+        /// </param>
+        /// <returns></returns>
+        public virtual IEnumerable<DataTransfer> GetTransfers(
+            IEnumerable<StorageTransferStatus> filterByStatus = default)
```

Comment:
I don't think this matters much in this scenario but I think it may be better to take in some already enumerated type here like `List<StorageTransferStatus>` or even `StorageTransferStatus[]`. Because you are calling `Contains` on this below, this will be enumerated once for each element in `_dataTransfers`. If it was a list type, then it wouldn't need to enumerate each time (just iterate). But again, probably doesn't matter in this small case.

I could also be off here. My knowledge of `IEnumerable` is a bit rusty for sure.